src/cache.py

```diff
@@ -0,0 +1,55 @@
+# |==========================================|
+# | Copyright (c) 2017-present Reece Dunham  |
+# | All rights reserved                      |
+# | Used with explicit permission            |
+# |==========================================|
+
+# Taken from a project that may become open source soon, please keep the copyright header
+
+class FileCache:
+    def __init__(self) -> None:
+        self.file_caches = []
+
+    def refresh(self, file: str) -> object:
+        with open(file, mode="r") as filehandler:
+            if not type(filehandler) is TextIOWrapper:
+                raise OSError("File handler not instance of TextIOWrapper! Quitting...")
+            else:
+                return list(filter(None, filehandler.read().split("\n")))
+
+    def get(self) -> object:
+        return self.file_caches
+
+    def get_by_int(self, num: int) -> object:
+        try:
+            return self.file_caches[num]
+        except Exception as e:
+            return e
+
+    def add(self, i: object) -> bool:
+        try:
+            self.file_caches.append(i)
+        except:
+            return False
+        finally:
+            return True
+
+    def clear_file(file: str) -> None:
```

Comment:
```suggestion
    def clear_file(self, file: str) -> None:
```
`TypeError: clear_file() takes 1 positional argument but 2 were given`